snakemake_conda/Snakefile

```diff
@@ -0,0 +1,7 @@
+rule run_python_script:
+    output:
+        "output_file.txt"
+    conda:
+        "env.yaml"
+    script:
+        "scripts/sample_script.py"
```

Comment:
I was going to suggest making this name more descriptive also, but now that I looked at the script I see it operates on random data. So maybe as another issue replace this sample rule with a script that reads our input data and outputs statistics.

Reply:
The new rule added now takes our actual input data and creates training data.